modules/programs/git.nix

```diff
@@ -41,10 +41,13 @@ in
         enable = mkEnableOption "Git";
 
         package = lib.mkPackageOption pkgs "git" {
+          nullable = true;
           example = "pkgs.gitFull";
           extraDescription = ''
             Use {var}`pkgs.gitFull`
             to gain access to {command}`git send-email` for instance.
+
+            Set to `null` to use the git not managed by home-manager.
```

Comment:
```suggestion
```

We don't need extra description for that, we have nullable's all throughout the codebase.